src/construction.cpp

```diff
@@ -978,7 +978,10 @@ bool can_construct( const construction &con, const tripoint &p )
     // see if the flags check out
     place_okay &= std::all_of( con.pre_flags.begin(), con.pre_flags.end(),
     [&p]( const std::string & flag ) {
```

Comment:
```suggestion
    [&p, &here]( const std::string & flag ) {
```
it's possible to directly reference map from outer scope when using lambda.